lib/travis/build/bash/travis_artifacts_install.bash

```diff
@@ -2,6 +2,10 @@ travis_artifacts_install() {
   local source="https://s3.amazonaws.com/travis-ci-gmbh/artifacts/stable/build/${TRAVIS_OS_NAME}/${TRAVIS_ARCH}/artifacts"
   local target="${TRAVIS_HOME}/bin/artifacts"
 
+  if [[ ${TRAVIS_OS_NAME} = "windows" ]]; then
```

Comment:
```suggestion
  if [[ ${TRAVIS_OS_NAME} == "windows" ]]; then
```
fixes `shfmt` violation which is what causes build failure. Technically, they are equivalent, but...

Reply:
Thanks for catching that.